nix/tools/google-java-format/default.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, fetchurl, jdk11, pkgs }:
```

Comment:
I assume it really needs jdk11 and doesn’t run on jdk8?

Reply:
Unfortunately the binary distribution is compiled to target JRE 11 and above and fails with the following if I try to use a JRE 8:

```
> javafmt
[dev-env] Building tools.javafmt...
[dev-env] Built tools.javafmt in /nix/store/fy4y0msyswvr0rk53wrsz4k3gccy2xzr-google-java-format and linked to /home/ste/Projects/digital-asset/daml/dev-env/var/gc-roots/javafmt
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.UnsupportedClassVersionError: com/google/googlejavaformat/java/Main has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:757)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:468)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:419)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
	at sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:495)
```

Reply:
makes sense, just wanted to check if you couldn’t get the nix magic to work to use jdk8 or if it really doesn’t run on jdk8.